### ISSUE TITLE
Add timestamp of tweet to toot

### ISSUE THREAD

**Issue by gdunstone** [yogthos/mastodon-bot#31]
Can we get a feature that adds the timestamp of the tweet to the corresponding toot?

Something like "Tweeted at 2020-01-28 10:20"

**Comment by yogthos** [comment]
Yeah that's pretty easy, looks like I already pull out the timestamp [here](https://github.com/yogthos/mastodon-bot/blob/bfe12535849bb5a5df82000fee1e1d8f50596047/mastodon-bot.cljs#L143) when the tweet is parsed, so it's just a matter of sticking it in the post body.

**Comment by gdunstone** [comment]
Cheers, I don't know Clojure that well, otherwise I would have made a pull request

**Comment by yogthos** [comment]
No worries, I'll try take a look in the near future.

**Comment by raboof** [comment]
I see the appeal, on the other hand the current behavior of 'all tweets since the latest toot' is pretty nice and predictable - I think I like it this way :)
